src/main/kotlin/no/nav/syfo/identhendelse/kafka/IdenthendelseConsumerService.kt

```diff
@@ -0,0 +1,50 @@
+package no.nav.syfo.identhendelse.kafka
+
+import no.nav.syfo.application.kafka.KafkaConsumerService
+import no.nav.syfo.identhendelse.IdenthendelseService
+import org.apache.avro.generic.GenericData
+import org.apache.avro.generic.GenericRecord
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.Duration
+
+class IdenthendelseConsumerService(
+    private val identhendelseService: IdenthendelseService,
+) : KafkaConsumerService<GenericRecord> {
+    override val pollDurationInMillis: Long = 1000
+
+    override fun pollAndProcessRecords(kafkaConsumer: KafkaConsumer<String, GenericRecord>) {
+        val records = kafkaConsumer.poll(Duration.ofMillis(pollDurationInMillis))
+        if (records.count() > 0) {
+            records.forEach { record ->
```

Comment:
Litt forenklet håndtering i forhold til i de andre app'ene, men tror dette er tilstrekkelig. Vil helst slippe å måtte legge inn en PDLClient i tillegg.

Reply:
Tror ikke jeg skjønner helt hvordan det vil gjøre at vi hopper over records :thinking: 
Da vil vi jo bare oppdatere til ny ident hos oss så lenge vi har noen forekomster på gammel ident. Hva er i så fall konskevensen av å ha noen forekomster hos oss på ny ident som ikke er oppdatert hos PDL?

Reply:
Nei, du har nok rett. Jeg er litt usikker på hva som skjer hvis vi har inconsistencies her. Det er kanskje verre i oversikten, hvis man åpner noen der, og så får man 404 fra syfoperson etterpå. Det kan ikke være snakk om mange sekunder/minutter det er feil, dog.

Reply:
Ja, det er akkurat det jeg tenker - her vil det jo alltid være "eventual consistency". Tror det er like greit med denne forenklingen for disse "perifere" app'ene (det verste som vil skje er jo at visningen av et vedtak vil være borte noen sekunder fra syfomodiaperson, dvs til alle app'ene har blitt oppdatert med den nye ident'en),